packages/model-viewer/src/three-components/SmoothControls.ts

```diff
@@ -823,23 +827,45 @@ export class SmoothControls extends EventDispatcher {
     const {isUserChange} = this;
     this.isUserChange = true;
 
-    switch (event.keyCode) {
-      case KeyCode.PAGE_UP:
+    // Handle modifier key conditions of:
+    // Shift & enablePan == PAN
+    // else == ORBIT/ZOOM
```

Comment:
Honestly, this logic is equally clear from reading the code, so I think we can skip this comment. Self-documenting code is ideal!